sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -281,6 +281,18 @@ extern "C" {
 #define CONFIG_GNRC_IPV6_NIB_NUMOF                   (4)
 #endif
 
+/**
+ * @brief Per-neighbor packet queue capacity
+ *
+ * If @ref CONFIG_GNRC_IPV6_NIB_QUEUE_PKT enabled, this is the maximum number
+ * of packets, per neighbor, awaiting packet resolution.
+ *
+ * @attention This MUST be leq UINT8_MAX
+ */
+#ifndef CONFIG_GNRC_IPV6_NIB_NBR_QUEUE_CAP
+#define CONFIG_GNRC_IPV6_NIB_NBR_QUEUE_CAP          (16)
```

Comment:
Any reason to have the default be so high?

Reply:
This has no effect when the neighbor count < 16 anyway, which is the default.